recipes/canmatrix/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "canmatrix" %}
+{% set version = "0.7dev" %}
+{% set sha256 = "d06dd8849fba6417d48639454221728f6c2779f17746174438039d3e44a40073" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  
+  fn: 8ab2bc648ebeef5f9c6b7368d260cd9c1b38a7fc.zip
+  url: https://github.com/ebroecker/canmatrix/archive/8ab2bc648ebeef5f9c6b7368d260cd9c1b38a7fc.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - xlwt
+    - xlrd
+    - xlsxwriter
+    - pyyaml
+    - lxml
+    - future
+
+test:
+  imports:
```

Comment:
You don't have to test import all of the dependencies. You typically should just have to import the package, so:

```
test:
   imports:
      - canmatrix
```